src/containers/Sidebar/index.tsx

```diff
@@ -27,10 +31,21 @@ class Sidebar extends React.Component<SidebarProps, {}> {
         this.props.requestMetadata();
     }
 
-    public onNavTabChange(selection: NavigationTab) {
+    public onNavTabChange(selection: string) {
         this.props.selectNavTab(selection);
     }
 
+    public renderMenu() {
+        if (this.props.menuData.length === 0) {
+            return null;
+        }
+        return (
+            <Menu
```

Comment:
Small stylistic nit: you don't have to do multiline JSX until you have more than one prop.